examples/common/avb.h

```diff
@@ -112,11 +112,11 @@ typedef enum { false = 0, true = 1 } bool;
 
 int pci_connect(device_t * igb_dev);
 
-int gptpscaling(gPtpTimeData * td, char *memory_offset_buffer);
+int gptpscaling(char *igb_mmap, gPtpTimeData *td);
```

Comment:
I used grep, and it seems like only  jackd_talker and simple_talker used it, and it does compile without problems.